mods/creative/locale/creative.id.tr

```diff
@@ -1,6 +1,6 @@
 # textdomain: creative
-Allow player to use creative inventory=Bolehkan pemain memakai inventaris kreatif
-No items to show.=
+Allow player to use creative inventory=Bolehkan pemain menggunakan inventaris kreatif
+No items to show.=Tiada barang untuk ditampilkan.
```

Comment:
I would usually use 'tidak ada' instead of 'tiada', since it sounds more poetic

Reply:
Yang mana yang lebih puitis? Kata _tiada_?

Awalnya, _tiada_ dipilih karena keterbatasan ruang. Cuman, kayaknya masih ada cukup ruang di versi yang terakhir.